assets/images/maps/madra/madra.json

```diff
@@ -317,8 +317,9 @@
-    },
+     "npc_04":"{\n    \"key_name\": \"foo\",\n    \"initial_x\": 25,\n    \"initial_y\": 15,\n    \"storage_keys\":{\n        \"affected_by_reveal\": \"herb_box_madra\",\n        \"visible\": \"herb_box_madra\"\n    },\n    \"npc_type\": \"sprite\",\n    \"movement_type\": \"idle\",\n    \"animation_key\": \"hidden_item\",\n    \"sprite_misc_db_key\": \"psynergy_particle\",\n    \"initial_action\": \"psynergy_particle\",\n    \"affected_by_reveal\": true,\n    \"events\": [\n        {\n            \"type\": \"branch\",\n            \"condition\": \"=\",\n            \"has_else\": true,\n            \"left_comparator_value\": {\n                \"type\": \"storage\",\n                \"value\": {\n                    \"key_name\": \"herb_box_madra\"\n                }\n            },\n            \"right_comparator_value\": {\n                \"type\": \"value\",\n                \"value\": true\n            },\n            \"events\": [{\n                \"type\": \"chest\",\n                \"custom_init_text\": \"${HERO} checked the wooden box...\",\n                \"item\": \"herb\",\n                \"no_chest\": true,\n                \"hide_on_finish\": true,\n                \"finish_events\": [{\n                    \"type\": \"set_value\",\n                    \"check_npc_storage_values\": true,\n                    \"event_value\": {\n                        \"type\": \"storage\",\n                        \"value\": {\n                            \"key_name\": \"herb_box_madra\",\n                            \"value\": false\n                        }\n                    }\n                }]\n            }],\n            \"else_events\": [{\n                \"type\": \"dialog\",\n                \"text\": \"${HERO} checked the wooden box...${BREAK}but didn't find anything.\"\n            }]\n        }\n    ]\n}",
+     "npc_05":"{\n    \"key_name\": \"psynergy_stone\",\n    \"storage_keys\":{\n        \"direction\": \"psynergy_stone\"\n    },\n    \"initial_x\": 33,\n    \"initial_y\": 15,\n    \"npc_type\": \"normal\",\n    \"movement_type\": \"idle\",\n    \"animation_key\": \"normal\",\n    \"base_collision_layer\": 0,\n    \"events\": [\n        {\n            \"type\": \"psynergy_stone\"}]\n}"
```

Comment:
Set `"storage_keys"` to:
```javascript
"storage_keys":{
    "active": "psy_stone_madra_active"
}
```

In the psynergy event section, set the `"finish_events"` to:
```javascript
"finish_events": [{
    "type": "set_value",
    "event_value": {
        "type": "storage",
        "value": {
            "key_name": "psy_stone_madra_active",
            "value": false
        }
    }
}]
```

Also, make sure the indentation of the whole event is ok when viewing it at Tiled.